src/views/AnalysisModule/Analysis/AnalysisPillar/index.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useCallback, useState } from 'react';
+
+import Icon from '#rscg/Icon';
+import QuickActionButton from '#dui/QuickActionButton';
+import Container from '#dui/Container';
+import Tag from '#dui/Tag';
+import FormattedDate from '#rscv/FormattedDate';
+import TextOutput from '#components/general/TextOutput';
+
+import {
+    AnalysisPillars,
+} from '#typings';
+
+import _ts from '#ts';
+import styles from './styles.scss';
+
+interface ComponentProps extends Omit<AnalysisPillars, 'id' | 'analysis'> {
+    pillarId: AnalysisPillars['id'];
+    setDeleteId: (value: number) => void;
+    createdOn: string | number;
+}
+
+function AnalysisPillar(props: ComponentProps) {
+    const {
+        pillarId,
+        title,
+        assigneeName,
+        setDeleteId,
+        createdOn,
+    } = props;
+
+    const [isCompleted, setIsCompleted] = useState(false);
```

Comment:
```suggestion
    const [completed, setCompleted] = useState(false);
```
Prefer not prefixing `is` on booleans